telegram/ext/choseninlineresulthandler.py

```diff
@@ -70,6 +74,11 @@ class ChosenInlineResultHandler(Handler[Update]):
             DEPRECATED: Please switch to context based callbacks.
         run_async (:obj:`bool`): Determines whether the callback will run asynchronously.
             Defaults to :obj:`False`.
+        pattern (:obj:`str` | `Pattern`, optional): Regex pattern. If not :obj:`None`, ``re.match``
+            is used on :attr:`telegram.ChosenInlineResult.result_id` to determine if an update
+            should be handled by this handler.
```

Comment:
Maybe add another sentence saying that the match will be available through `context`? Apparently we don't have that for e.g. `CallbackQueryHandler`, but that shouldn't stop us :)

Reply:
I will add it to the doc-fixes PR for the CallbackQueryHandler

Reply:
good idea :)

Reply:
done: e3745fa90971fd250984fa1282192aafda3eb5e0